### ISSUE TITLE
Add ability to make requests in ldap

### ISSUE THREAD

**Issue by Lichtsinnig** [Jeffail/benthos#380]
Add the ability to make requests in ldap, similar to the http module.
This will enrich the flow data with data from Ldap and resolve the object SID data

**Comment by moreda** [comment]
Hi!

Is there any update on this? 😊

I have a case where a LDAP input is needed. I guess it is not uncommon, so just asking in case someone else is in the same spot. It does not seem a complex issue (famous opening words 😊) and I would consider a mini-development instead of using a subprocess. The subprocess path involving ldapsearch adds a lot of messy post-processing due to the ldif output format, so it breaks the elegance of the solution.

**Comment by moreda** [comment]
Yes, I was thinking about that. ATM I am a bit swamped with work but let me see in a couple of weeks. Thx!

**Comment by twmb** [comment]
LDAP support is a niche use case. The go-ldap library provides a good foundation for an external plugin. This has been open since 2020 with no implementation, so we're closing it for now. Feel free to open a new issue or PR if this is still needed.
